{"name": "test_right_way", "status": "skipped", "statusDetails": {"message": "XFAIL Wait for fix bug\n\nselenium.common.exceptions.TimeoutException: Message:", "trace": "tests\\test_shop.py:90: in test_right_way\n    WebDriverWait(browser, timeout=10, poll_frequency=1).until(\n        browser    = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"e39a2a73ee3e73778bd133d9a199acdd\")>\n        cart_is_visible = 'block'\n        product    = <selenium.webdriver.remote.webelement.WebElement (session=\"e39a2a73ee3e73778bd133d9a199acdd\", element=\"5533bf34-2736-494d-bf74-0c3b095921f6\")>\n..\\..\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:90: in until\n    raise TimeoutException(message, screen, stacktrace)\nE   selenium.common.exceptions.TimeoutException: Message:\n        end_time   = 28697.062\n        message    = ''\n        method     = <function url_to_be.<locals>._predicate at 0x000001111C3955A0>\n        screen     = None\n        self       = <selenium.webdriver.support.wait.WebDriverWait (session=\"e39a2a73ee3e73778bd133d9a199acdd\")>\n        stacktrace = None\n        value      = False"}, "description": "\n    Test case TC-4\n    ", "start": 1666802601263, "stop": 1666802693625, "uuid": "5f010917-651b-4701-ac3a-3b25aeb17eb5", "historyId": "832350b147fac251f172289a626c5cba", "testCaseId": "7b8323124234f69a8fde1e3de50f452e", "fullName": "tests.test_shop#test_right_way", "labels": [{"name": "tag", "value": "@pytest.mark.xfail(reason='Wait for fix bug')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_shop"}, {"name": "host", "value": "DESKTOP-NDLNMNH"}, {"name": "thread", "value": "8264-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_shop"}]}